.circleci/config.yml

```diff
@@ -69,11 +69,15 @@ jobs:
       - run:
           name: Run unit tests
           command: yarn test --runInBand
+      - save_workspace
```

Comment:
I guess my mental model is:

- deploy relies on test relies on build
- build saved the workspace
- test did not
- maybe test needs to save the workspace too?

Reply:
No, test doesn't blow away the dist/; I was mistaken. The problem was that the dist/ was never created. :)